template/pyproject.toml.jinja

```diff
@@ -117,29 +114,7 @@ lint.select = [
     "SIM", # flake8-simplify
 ]
 lint.ignore = [
```

Comment:
This should be moved to `tool.ruff.lint` section to be more readable (and consistent with the napari main repository).